packages/es-ingest/src/main/resources/application-local.properties

```diff
@@ -5,20 +5,10 @@ terarium.elasticsearch.url=http://localhost:9200
 terarium.elasticsearch.auth-enabled=false
 
 ########################################################################################################################
-# Ingest configuration
+# aws credentials configuration
 ########################################################################################################################
-terarium.esingest.ingestParams[0].name="Model Dataset"
-terarium.esingest.ingestParams[0].inputDir=/home/kbirk/Downloads/model
-terarium.esingest.ingestParams[0].topics=model, amr
-terarium.esingest.ingestParams[0].outputIndexRoot=searchable_model
-terarium.esingest.ingestParams[0].ingestClass=software.uncharted.terarium.esingest.ingests.model.ModelIngest
-terarium.esingest.ingestParams[0].clearBeforeIngest=true
-terarium.esingest.ingestParams[0].ignoreMissingDocumentsDuringUpdates=true
-
-terarium.esingest.ingestParams[1].name="Opensource Epidemiology Dataset"
-terarium.esingest.ingestParams[1].inputDir=/home/kbirk/Downloads/epidemiology
-terarium.esingest.ingestParams[1].topics=climate, epidemiology
-terarium.esingest.ingestParams[1].outputIndexRoot=searchable_document
-terarium.esingest.ingestParams[1].ingestClass=software.uncharted.terarium.esingest.ingests.document.DocumentIngest
-terarium.esingest.ingestParams[1].clearBeforeIngest=true
-terarium.esingest.ingestParams[1].ignoreMissingDocumentsDuringUpdates=false
+terarium.file-storage-s3-bucket-name=askem-local-storage
+terarium.amazon.credential.access-key=admin
+terarium.amazon.credential.secret-key=admin123
+terarium.amazon.s3.region=us-east-1
+terarium.amazon.s3.url=http://localhost:9000
```

Comment:
I believe this is supposed to be `minio` not `localhost` for running locally (within Docker-world).  I have taken to adding an `application-intellij.properties` so that I can override `minio` to `localhost` and not have a /etc/hosts entry for that :) - Currently that property file is in a PR of mine - 2702

Reply:
Ooo okay I was wondering. Mind renaming it to `application-ide.properties` so us vscode plebs don't feel left out? :)

Reply:
Added it to this PR actually.